modules/remove.js

```diff
@@ -0,0 +1,8 @@
+// removes first element having the condition
+export default function remove(collection, predicate) {
+  var index = collection.length;
```

Comment:
... which is to pass the predicate through our internal `cb` function. This enables the iteratee shorthands, so users can for example do `_.remove(anArray, { a: 1 })` (which would be equivalent to your testcase where you're passing `function(obj) { return obj.a == 1; }` as the second argument).

```suggestion
  var index = collection.length;
  predicate = cb(predicate);
```